blockonomics.py

```diff
@@ -83,6 +83,38 @@ def check_for_pending_transactions(self) -> None:
             )
 
     def _get_history_for_addresses(self, addresses: list) -> dict:
+
+        debug = configloader.user_cfg["Bitcoin"].get('debug', False)
```

Comment:
This throws an error if the config doesn't have `debug` variable. I think it should still continue with debug as false if not present.